test/cases/api.sh

```diff
@@ -89,6 +94,7 @@ case $CLOUD_PROVIDER in
     checkEnvAzure
     ;;
 esac
+checkEnvSubscription
```

Comment:
Move to the case statement above and check only RHEL runners.

Reply:
Thanks! Moved to RHEL only.